app/workflows/initial_tasks_factory.rb

```diff
@@ -36,6 +36,9 @@ def create_vso_tracking_tasks
       if TrackVeteranTask.where(appeal: @appeal, assigned_to: rep).empty?
         TrackVeteranTask.create!(appeal: @appeal, parent: @root_task, assigned_to: rep)
       end
+      if ChangeHearingRequestTypeTask.where(appeal: @appeal, assigned_to: rep).empty?
```

Comment:
This conditional logic needs to be updated to only create the ChangeHearingRequestTypeTask if `@appeal.docket_type == "hearing"` (See [AC 1.1](https://vajira.max.gov/browse/CASEFLOW-1520))

```suggestion
      if ChangeHearingRequestTypeTask.where(appeal: @appeal, assigned_to: rep).empty? &&
         @appeal.docket_type == "hearing"
```